_overviews/toolkit/sttp-json.md

```diff
@@ -0,0 +1,92 @@
+---
+title: How to send and receive JSON?
+type: section
+description: How to send JSON in a request and to parse JSON from the response.
+num: 27
+previous-page: sttp-request-body
+next-page: sttp-upload-file
+---
+
+{% include markdown.html path="_markdown/install-sttp.md" %}
+
+## HTTP and JSON
+
+JSON is common format to send and receive data through HTTP, in the request and response bodies.
+
+In the examples below, We use the [Github REST API](https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28).
+You will need a secret [Github authentication token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) to run the programs.
+Do not share this token with anyone and do not paste it online!
+
+## Sending and receiving JSON
+
+To send JSON in a request and parse JSON from a response we use sttp in combination with uJson.
+
+### Sending JSON
+
+To send JSON, you can construct a `uJson.Value` and write it as a string in the body of the request.
+
+In the following example we use [Github users endpoint](https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28) to update the profile of the authenticated user.
+We provide the new location and bio of the profile in a JSON object.
+
+{% tabs 'json' %}
+{% tab 'Scala 2 and 3' %}
+```scala
+import sttp.client4.quick.*
+
+val json = ujson.Obj(
+  "location" -> "hometown",
+  "bio" -> "Scala programmer"
+)
+
+val response = quickRequest
+  .patch(uri"https://api.github.com/user")
+  .auth.bearer(sys.env("GITHUB_TOKEN"))
+  .header("Content-Type", "application/json")
```

Comment:
it might be worth pointing out that headers should go before the body, because otherwise this would add a second `Content-Type` header